app/models/manageiq/providers/openstack/cloud_manager/orchestration_template.rb

```diff
@@ -0,0 +1,131 @@
+class ManageIQ::Providers::Openstack::CloudManager::OrchestrationTemplate < ::OrchestrationTemplate
```

Comment:
This is a class moved from core repository https://github.com/ManageIQ/manageiq/blob/master/app/models/orchestration_template_hot.rb#L1

The old file will be removed after the migration is done.